### PR TITLE
added basic css structure

### PR DESCRIPTION
**Pull request by phillip-phan-imprivata**
added flex to containers and sizing for images

```
git fetch --all
git checkout pp-francepage
serve
```

### CODE REVIEW

**Review by Younique98 — COMMENTED** [review]
This looks good

**Review by TrentSisson — APPROVED** [review]
good to go once we resolve merge issues

**Pull request merged**
